apps/kyb-app/src/pages/CollectionFlow/versions/v2/components/organisms/CollectionFlowUI/hooks/usePriorityFields/utils/generate-priority-fields.ts

```diff
@@ -26,20 +27,27 @@ export const generatePriorityFields = (
         const documents = get(
           context,
           formatValueDestination(element.valueDestination, stack),
-        ) as IDocumentTemplate[];
+        ) as Array<IDocumentTemplate<IDocumentRecord>>;
         const document = documents?.find(
           (doc: IDocumentTemplate) => doc.id === element.params?.template?.id,
         );
 
-        const reason = document?.decisionReason;
+        const isRevisionOrRequested =
```

Comment:
This section might introduce regression to users who use both documents and collection flow